src/Sarif.Multitool.Library/Rules/Base1001MaximumRunsCount.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
+{
+    public class Base1001MaximumRunsCount : SarifValidationSkimmerBase
+    {
+        public override string Id => string.Empty;
+
+        public override MultiformatMessageString FullDescription => new MultiformatMessageString();
+
+        public virtual int MaximumRuns { get; set; }
+
+        protected override void Analyze(SarifLog sarifLog, string runPointer)
+        {
+            if (sarifLog.Runs?.Count > this.MaximumRuns)
```

Comment:
>sarifLog.Runs?.Count > this.MaximumRuns [](http://example.com/codeflow?start=16&length=39)

I think when Runs is null, this will not be true

Reply:
Yeah, and that's correct. Null == 0.